pyof/v0x04/asynchronous/error_msg.py

```diff
@@ -197,6 +197,30 @@ class ErrorType(IntEnum):
     #: Experimenter error messages.
     OFPET_EXPERIMENTER = 0xffff
 
+    def get_class(self):
+        """Return a Code class based on current ErrorType value.
+
+        Returns:
+            enum.IntEnum: class referenced by current error type.
+
+        """
+        classes = {'OFPET_HELLO_FAILED': HelloFailedCode,
+                   'OFPET_BAD_REQUEST': BadRequestCode,
+                   'OFPET_BAD_ACTION': BadActionCode,
+                   'OFPET_BAD_INSTRUCTION': BadInstructionCode,
+                   'OFPET_BAD_MATCH': BadMatchCode,
+                   'OFPET_BAD_REQUEST': BadRequestCode,
+                   'OFPET_FLOW_MOD_FAILED': FlowModFailedCode,
+                   'OFPET_GROUP_MOD_FAILED': GroupModFailedCode,
+                   'OFPET_PORT_MOD_FAILED': PortModFailedCode,
+                   'OFPET_QUEUE_OP_FAILED': QueueOpFailedCode,
+                   'OFPET_SWITCH_CONFIG_FAILED': SwitchConfigFailedCode,
+                   'OFPET_ROLE_REQUEST_FAILED': RoleRequestFailedCode,
+                   'OFPET_METER_MOD_FAILED': MeterModFailedCode,
+                   'OFPET_TABLE_MOD_FAILED': TableModFailedCode,
+                   'OFPET_TABLE_FEATURES_FAILED': TableFeaturesFailedCode}
+        return classes[self.name]
```

Comment:
Would it be worth to change `return classes[self.name]` to `return classes.get(self.name, SomeGenericDefaultClass)`, or is this a bad idea?

Reply:
Good point @hdiogenes . IMO, a generic exception in this case doesn't add that much valuable for the end user. I believe having an explicit crash will force the user to report a bug if for reason reason another unexpected issue arises. @beraldoleal, do you have any suggestion/preference about this?

Reply:
@hdiogenes's suggestion is good. It will avoid a KeyError and will not impact on user point of view, since that it will return the specific error message.

Please @viniarck can you change that?